template.rb

```diff
@@ -166,20 +169,40 @@ def create_bin_bundle
   chmod "bin/bundle", "+x"
 end
 
-def setup_adrs
-  return say('ADRs already setup, skipping') if file_contains?('Gemfile', 'rladr')
-  say("\n=== Architecture Decision Records (ADRs) ===")
-  return unless yes?('Add `rladr` for Architecture Decision Record (ADR) support? y/N')
 
-  apply 'adr/template.rb'
+def get_tools_version_of(tool_name)
+  File.readlines(".tool-versions")
+    .map(&:split)
+    .find { |tool, version| tool == tool_name }
+    .[](1)
```

Comment:
I think `.at(1)` here might be easier to understand than passing an arg to square brackets.

Reply:
💯  ... thanks for that, I was clearly just too lazy to find the correct, idiomatic method to use there. Fixed up in the original commit to try to keep things (somewhat) tidy.

Reply:
Nice! I didn't realise `[]` could take arguments, that was an education in itself 🙂